components/Form.js

```diff
@@ -73,6 +73,7 @@ export default function Form() {
         type="text"
         placeholder="FIRST"
         aria-label="input for the first keyword"
+        autocomplete="chrome-off"
```

Comment:
I don't know this attribute 🤔 

This looks more like it:
```suggestion
        autocomplete="off"
```

Reply:
it didnt do what i wanted anyway, forgot to delete it